x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/styles.tsx

```diff
@@ -72,56 +86,59 @@ export const LabelNodeContainer = styled.div`
 `;
 
 interface LabelShapeProps extends EuiTextProps {
-  color: LabelNodeViewModel['color'];
+  backgroundColor?: string;
+  borderColor?: string;
+  shadow?: string;
 }
 
-export const LabelShape = styled(EuiText)<LabelShapeProps>`
-  background: ${(props) => useNodeFillColor(props.color)};
+export const LabelShape = styled(EuiText, {
+  shouldForwardProp(propName) {
+    return !['backgroundColor', 'borderColor', 'isConnectable', 'shadow'].includes(propName);
+  },
+})<LabelShapeProps>`
+  background-color: ${(props) => props.backgroundColor};
+  border: ${(props) => `${LABEL_BORDER_WIDTH}px solid ${props.borderColor}`};
   max-width: ${NODE_LABEL_WIDTH - LABEL_PADDING_X * 2 - LABEL_BORDER_WIDTH * 2}px;
   max-height: ${NODE_LABEL_HEIGHT - LABEL_BORDER_WIDTH * 2}px;
-  border: ${(props) => {
-    const { euiTheme } = useEuiTheme();
-    return `solid ${
-      euiTheme.colors[props.color as keyof typeof euiTheme.colors]
-    } ${LABEL_BORDER_WIDTH}px`;
-  }};
-
-  font-weight: ${(_props) => {
-    const { euiTheme } = useEuiTheme();
-    return `${euiTheme.font.weight.semiBold}`;
-  }};
-  font-size: ${(_props) => {
-    const { euiTheme } = useEuiTheme();
-    return `${euiTheme.font.scale.xs * 10.5}px`;
-  }};
 
   line-height: 1.5;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 
   padding: 6px ${LABEL_PADDING_X}px;
-  border-radius: 16px;
+  border-radius: ${LABEL_BORDER_RADIUS}px;
   min-height: 100%;
   min-width: 100%;
+
+  ${({ shadow }) => `
```

Comment:
nit I think it is safe to remove shadow property.
We want to keep the style of the shadow consistent. I do see any reason to expose it as a property